content/en/smart-contracts/fundamentals/aggregated-deal-making/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Aggregated deal-making"
+description: "Learn about Aggregated Deal-making on the Filecoin blockchain, where developers can combine small storage deals into larger, more attractive deals for storage providers."
+lead: "Filecoin is designed to store large data for extended periods. Small-scale data (<4 GiB) can be combined with other small deals into larger ones, either on-chain or off-chain. Smart contracts can handle programmatic data storing. This article explains the process, referring to small-scale data as _sub-piece data_."
+draft: false
+images: []
+type: docs
+menu:
+  smart-contracts:
+    parent: "smart-contracts-fundamentals"
+    identifier: "aggregators"
+weight: 192
+toc: true
+---
+
+Aggregation is the process of combining multiple packages of _sub-piece data_ into a single package. This large package is stored on the Filecoin network instead of multiple smaller packages. Aggregation can be done off-chain or on-chain.
```

Comment:
We should probably define what a subpiece is, explain why the Filecoin network cannot store subpieces as-is, and give examples of common subpieces (a video, NFT art, small dataset, etc.)